app/src/main/java/com/github/se/polyfit/model/meal/Meal.kt

```diff
@@ -0,0 +1,88 @@
+package com.github.se.polyfit.model.meal
+
+import com.github.se.polyfit.model.ingredient.Ingredient
+
+// modeled after the log meal api
+data class Meal(
+    val occasion: MealOccasion,
```

Comment:
maybe I'm looking ahead too far, but we can potentially add an optional temperature field to Meal for in the future suggesting food based on similar temperatures

Reply:
I attempted to model the classes based on the API results, but I could definitely include that. Were you thinking more along the lines of a range or an enum for temperature, like cold, medium, hot?

Reply:
i had in mind an integer representing temp in celsius so we can have more precision control, but open to either